assignments/prototypes.js

```diff
@@ -35,14 +39,32 @@ function GameObject(attributes) {
 function CharacterStats (stats){
   this.hp = stats.hp;
   this.name = stats.name;
-  this.takeDamage = function () {
-    CharacterStats.call(destroy);
-    return `${this.name} took damage.`;
-  }
+  
+}
+function takeDamage () {
+  alert (`${this.name} took damage.`);
 }
+takeDamage.call(CharacterStats);
+destroy.call(CharacterStats);
```

Comment:
Using:

>CharacterStats.prototype = Object.create(GameObject.prototype);

and
>GameObject.call(this, stats);

should be all you need to 'inherit' the `destroy` method and have access to GameObject's 'this'. You shouldn't have to use `.call` on destroy. `takeDamage` will be bound to CharacterStats similarly to `destroy`'s binding to GameObject.